README.md

```diff
@@ -129,8 +129,8 @@ Release
    - bump the version in `pynvim/util.py` and `setup.py` (3 places in total)
 2. Make a release on GitHub with the same commit/version tag and copy the message.
 3. Run `scripts/disable_log_statements.sh`
-4. Run `python setup.py sdist`
+4. Run `python -m build`
```

Comment:
This seems to be only for a particular PEP517 choice [python-build](https://github.com/pypa/build). Shouldn't we have `pip install build` somewhere?

Reply:
I hadn't thought of adding that, but will if you believe it's necessary.  I don't think it's really necessary for two reasons:

1. This is only for building the package for distribution - not for general end-user installation or re-installation.  The `build` package is not actually required for installing our package.
2. We're using the twine library a few lines below and don't mention `pip install twine` for that either.

I chose this method because it brings us into compliance with pep 517 with minimal effort and almost no change to the package as a whole.

At some point, someone _will_ probably want to do some more work to move a lot of the code from the `setup.py` file to the `pyproject.toml` file, but this at least makes us compliant and will stop pip from complaining every time someone installs the package (as it is doing now).